### DISCUSSION TITLE
VScode is using brave for how to change it

### DISCUSSION THREAD

**Discussion by ailakshya** [prasanthrangan/hyprdots#1862] [Q&A]
![tempImageGaJRhi](https://github.com/user-attachments/assets/fa9dc1fb-8f72-41a9-b5ce-b9e28a95d99c)
which gives me the broken tockens for my extensions

**Comment by kRHYME7** [comment]
Just choose the blowfish encryption

**Reply by thyeun** [reply]
if you dont want using ```kdewallet```, you can disable it.

Just go into ```kwalletrc``` at ```~/.config```. With the below setting.

```
[Wallet]
First Use=false
Enabled=false
```
